README.md

```diff
@@ -85,6 +85,53 @@ it will be downloaded and installed in the source tree.
 #### Debian-based distro:
 
     sudo apt-get install nodejs git build-essential python2.7
+    #Raspberry Pi Cjdns Install
+
+##### Raspberry Pi
```

Comment:
Seems fixed. I linked to the repository instead of the script. Give people for information.